File: beecrowd/go/bee1117.go
package main

import "fmt"

func main() {

	var input, avg float64 = 0, 0

	counter := 0
	for counter < 2 {
		fmt.Scanf("%f", &input)

		if !(input < 0 || input > 10) {
			avg += input
			counter++
		} else {
			fmt.Println("nota invalida")
		}

	}

	fmt.Printf("media = %.2f\n", (avg / 2))

}

File: beecrowd/c/bee1117.c
#include <stdio.h>

int main() {

  int counter = 0;
  float input, avg = 0.0f;

  while (counter < 2) {
    scanf("%f", &input);

    if (!(input < 0 || input > 10)) {
      avg += input;
      counter++;
    } else {
      printf("nota invalida\n");
    }
  }

  printf("media = %.2f\n", (avg/counter));

  return 0;
}
